packages/db/tests/search/test-ExecutionSearch.js

```diff
@@ -145,6 +147,7 @@ test('ExecutionSearch returns correct response for basic query', async (t) => {
 test('ExecutionSearch supports page and limit params', async (t) => {
   const { knex } = t.context;
   let queryStringParameters = {
+    estimateTableRowCount: 'false',
```

Comment:
For execution and granules, estimateTableRowCount is set to true by default, unless it's passed in as false.  I set them to false in these tests, so that the accurate count is returned.